src/test/ts/part2/Exercise3OptionTest.ts

```diff
@@ -12,7 +12,9 @@ describe('Exercise3OptionTest', () => {
   });
 
   it('toPositiveInteger', () => {
-    // TODO: write a few test cases
+    assert.deepEqual(Ex.toPositiveInteger(2).getOrDie(), 2)
```

Comment:
I notice that you don't use `;` at the end of lines. We use that for the ts code but we have eslint tools to enforce that this project doesn't use our standard set of eslint rules. But I guess it would be a bit more distracting if you would have to deal with that while also learning the other things and concepts so not sure if this project should have those lint tools or not. I will check that with some other people.

Reply:
Oh I never pay attention to those because I usually have format on save enabled in my editor, so the aslant/prettier extensions would do that for me. I think having a `.eslintrc` and `.prettierrc` files would help